### PR TITLE
[CDAP-5395] Feature/ui logs quote fix

### PR DESCRIPTION
**Pull request by elfenheart**
- Fix log viewer to not escape html
- Fix hydrator list view status for DataPipeline

### CODE REVIEW

File: cdap-ui/app/directives/log-viewer/log-viewer.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ angular.module(PKG.name + '.commons')
           params = {};
           angular.copy($scope.params, params);
           params.max = 50;
+          params.escape = false;
```
Comment (anwar6953) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: who uses this value?

Comment (elfenheart) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: The API call. That becomes the query parameter for logs

Comment (anwar6953) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: this is not redundant, compared to your change on line 70?
Why are both necessary?

Comment (elfenheart) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: No it's not. The one is for polling data, and the other is for $resource which we use after the poll of the logs have stopped. Currently we poll for data until there is a fetch for next set of logs.

Comment (anwar6953) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: got it.

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**